cmd/appstate.go

```diff
@@ -40,7 +40,10 @@ func (a *appState) loadConfigFile(ctx context.Context) error {
 
 	if _, err := os.Stat(cfgPath); err != nil {
 		// don't return error if file doesn't exist
-		return nil
+		if os.IsExist(err) {
```

Comment:
small nit but the [Go docs](https://pkg.go.dev/os#IsExist) for `IsExists` states, "This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, fs.ErrExist)."

Could we possibly swap the `IsExists` call out for `errors.Is`?